org.pjbroad.EternallandsClient.desktop

```diff
@@ -0,0 +1,24 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Eternal Lands (flatpak)
```

Comment:
I would remove the " (flatpak)" suffix. It's not important (and can be confusing) to the end-user.

Reply:
Yep, this has largely been for my own benefit during development as I'm also developing the snap version and have other installs present too.  I'll take it out before finishing if this pull request is successful :)